_posts/2015-02-24-how-to-use-github-and-the-terminal-a-guide.md

```diff
@@ -0,0 +1,283 @@
+Title: How to Use Github and the Terminal: A Guide
+Author: melody, boone
+Layout: Page
+
+---
+
+
+This is a guide that assumes you have no prior knowledge of Github or the command line. We're going to introduce you to both Github and using the command line through a guided exercise. Today you'll learn how to make a blog post on the [18F blog](https://18f.gsa.gov/news/).
+
+Every step will be illustrated with a helpful screenshot or animated gif that shows you exactly what your screen should look like. We'll go through each step in order. At the end of this post, you will know how to:
+
+1. [Set Up A Cloned Repo On Your Desktop](TKTKT)
+2. [Create an SSH Key](TKTKT)
+3. [Read what's in your cloned directory](TKTKTTK)
+4. [Install a Package Manager](TKTKKTT)
+5. [Create a Branch](TKTKTKT)
+6. [Edit and Post a Blog Post](TKTKKTT)
+7. [Add Front Matter](TKTKTKT)
+8. [Learn how to make a pull request](TKTKT)
+
+It is worth noting: There are many different ways to do each of these steps. If you have an alternative way of doing any of these steps — or have ways to make this more efficient — please let us know by posting an issue [here](https://github.com/18f/18f.gsa.gov/issues/new). (You don't have to know how to code to post an issue, but you do need a Github account.) 
+
+**What you need to get started** a Github account, Terminal, Sublime or another text editing program. These instructions are for Macs and Linux-enabled machines. If you are working on Windows, we suggest checking out [this comment](https://github.com/18F/18f.gsa.gov/issues/542#issuecomment-75145417) that was posted to Github that details how to make these instructions work for Windows machines.
+
+### How to Set Up A Cloned Repo On Your Desktop ###
+
+* First, you want to log into Github. Go to [http://www.github.com/login](https://github.com/login) and sign in with your username and password. 
+
+* Go to Finder and type in Terminal. Open the terminal. 
+
+>Most of the next steps are things you only need to do once. It seems like a lot, but it's basically getting >everything in place so you can do this very easily. It will seem like a lot. It's one time only. Promise. There's >a little notification when you're done the one-time only stuff.
+
+* Type in `xcode-select --install.` This will install git along with other developer tools that Apple provides on your machine. (To learn more about this command, click [here](http://railsapps.github.io/xcode-command-line-tools.html)
+
+>This can take a very long time. Even though it's early in the process, might as well take a break or grab lunch. >Come back in an hour. But don't shut your computer off.**
+
+* Type in `ls`. This will show you everything in the directory where you are currently located.
+
+* You're going to want to create a new directory where you will clone all of your Github repos. This keeps everything nice and neat. You do this using the `mkdir` command, which stands for "Make Directory." I called my directory `Code.` 
+
```

Comment:
I would recommend making directory names all lowercase. It's easier to type, but since your animated GIFs have the capitalized version, it's fine.